components/activity-indicator/index.tsx

```diff
@@ -0,0 +1,128 @@
+import classNames from 'classnames';
+import * as React from 'react';
+import { ConfigContext } from '../config-provider';
+import useStyle from './style/index';
+
+export interface ActivityIndicatorProps {
+  prefixCls?: string;
+  size?: number;
+  color?: string;
```

Comment:
add param loading => show when this indication should appear and easier to handle display effect